env/Tensorflow_probability.yml

```diff
@@ -0,0 +1,33 @@
+name: Tensorflow
+channels:
+  - conda-main
+  - conda-mo-internal
+  - conda-forge
+  - conda-r
+dependencies:
+  - python=3.7
+  - tensorflow-probability=0.12
+  - ipykernel
+  - tensorboard=2.4
+  - notebook
+  - matplotlib
+  - keras-applications
+  - pydot
+  - graphviz
+  - cf_units
+  - cf-units=3.0.0
+  - iris=3.1.0
```

Comment:
i might suggest just having major and minor  version nums, and allowing the last bit to float. Usually any breaking changes should change the minor version number, and the last value should only be about bug fixes. So I'd suggest specifying iris=3.1 and so forth

Reply:
Good call